### ISSUE TITLE
Suggestion: mDNS

### ISSUE THREAD

**Issue by deanlyoung** [biemond/com.athom.awair#10]
We added mDNS for device discovery on the local network, which is available on 2nd Edition and Element. The URL would look like this: http:// + device_id + MAC NIC + .local ( e.g. http://awair-r2-123abc.local or http://awair-elem-456edf.local ). Might be easier to setup / use / discover instead of the IP address.

https://github.com/biemond/com.athom.awair/blob/60c1582b3eb1b0be2fe3620334aad7e4076548cc/drivers/localDevice/pair/pair.html#L22

**Comment by biemond** [comment]
great plan. let me try it out.

**Comment by biemond** [comment]
He , I try to find it but somehow I don't see it on my r2 or mint.

I see a lot on this mdns or bonjour explorer but I don;t see the awair devices.

**Comment by deanlyoung** [comment]
Are you using this tool?
https://apps.apple.com/us/app/discovery-dns-sd-browser/id305441017
![image](https://user-images.githubusercontent.com/2523456/103225145-2b389480-48de-11eb-824a-fe8df4d8f8a2.jpeg)

**Comment by biemond** [comment]
<img width="492" alt="Screen Shot 2020-12-28 at 16 32 55" src="https://user-images.githubusercontent.com/1085004/103225526-619c0100-492a-11eb-9c47-864eb52aa70f.png">
I am on the same wifi and vlan. I see a  lot but don't see these.  

what firmware are you using.

**Comment by biemond** [comment]
it looks like a reboot  helps.

**Comment by deanlyoung** [comment]
Oh, interesting... 🤔 haven't seen that before. Was it related to turning it on, and it needing to broadcast its multicast messages perhaps?

**Comment by biemond** [comment]
weird thing , I cannot connect the r2 again to awair, also the mdns broadcast is gone.  My mint is working, also after the reboot but I don't see this mdns broadcast.
already tried multiple networks with or without vlan.  I will shut it down for some time.

**Comment by deanlyoung** [comment]
Oh, I think I see a UniFi controller in the mDNS list. This feature has some known issues (not sure of the root cause) with Ubiquiti equipment. Something about the mDNS messages doesn't play nice...

**Comment by biemond** [comment]
yep, I don't see all the time in my vlan but this works from my vlan to the iot vlan. 
I made a support call to awair , I see the https traffic between awair and my  device but I guess something went wrong.
```
 ping AWAIR-R2-105560.local
PING awair-r2-105560.local (192.168.107.43): 56 data bytes
64 bytes from 192.168.107.43: icmp_seq=0 ttl=127 time=4.891 ms
64 bytes from 192.168.107.43: icmp_seq=1 ttl=127 time=4.251 ms
64 bytes from 192.168.107.43: icmp_seq=2 ttl=127 time=3.997 ms
```

**Comment by deanlyoung** [comment]
I've read that there are issues with mDNS broadcasts across vlans.

https://community.ui.com/questions/No-IP-but-able-to-connect-to-2-4GHZ-wifi-Awair-unit/7246c97e-326f-4131-adf0-c0b2b7d425f0

**Comment by biemond** [comment]
yep, tried some versions old unify versions, awair has probably some clues what it can be but still weird why broadcast is affecting awair 2nd so much and my mint is all good with this.

**Comment by deanlyoung** [comment]
The firmwares are slightly different and there isn't support for mDNS on Mint.
